vql/common/shell.go

```diff
@@ -121,19 +121,19 @@ func (self ShellPlugin) Call(
 
 		stdout_pipe, err := command.StdoutPipe()
 		if err != nil {
-			scope.Log("shell: no command to run")
+			scope.Error("shell: no command to run")
```

Comment:
This is probably the wrong message :-) 

IDK - is this critical or not? it might be or it might be intermittent (e.g. tool failed to download etc)